charts/app-config/values-production.yaml

```diff
@@ -1119,6 +1119,29 @@ govukApplications:
         - name: WEB_CONCURRENCY
           value: '4'
 
+  - name: draft-finder-frontend
+    repoName: finder-frontend
+    helmValues:
+      arch: arm64
```

Comment:
We don't have arm nodes in production yet

Reply:
Thanks - removed 👍